src/codemagic/tools/_app_store_connect/action_groups/beta_build_localizations_action_group.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import annotations
+
+from abc import ABCMeta
+from typing import List
+from typing import Optional
+
+from codemagic import cli
+from codemagic.apple.app_store_connect.versioning import BetaBuildLocalizations
+from codemagic.apple.resources import BetaBuildLocalization
+from codemagic.apple.resources import Locale
+from codemagic.apple.resources import ResourceId
+
+from ..abstract_base_action import AbstractBaseAction
+from ..action_group import AppStoreConnectActionGroup
+from ..arguments import BuildArgument
+
+
+class BetaBuildLocalizationsActionGroup(AbstractBaseAction, metaclass=ABCMeta):
+
+    @cli.action('list',
+                BuildArgument.BUILD_ID_RESOURCE_ID,
+                BuildArgument.LOCALE_OPTIONAL,
+                action_group=AppStoreConnectActionGroup.BETA_BUILDS_LOCALIZATIONS)
+    def list_beta_build_localization(
```

Comment:
Maybe use plural in method name?
```suggestion
    def list_beta_build_localizations(
```